cfgov/v1/models/learn_page.py

```diff
@@ -155,8 +197,8 @@ class EventPage(AbstractFilterPage):
         related_name='+'
     )
     flickr_url = models.URLField("Flickr URL", blank=True)
-    youtube_url = models.URLField("Youtube URL", blank=True,
-    help_text="Format: https://www.youtube.com/embed/video_id. It can be obtained by clicking on Share > Embed on Youtube.",
+    youtube_url = models.URLField("YouTube URL", blank=True,
+    help_text="Format: https://www.youtube.com/embed/video_id. It can be obtained by clicking on Share > Embed on YouTube.",
     validators=[ RegexValidator(regex='^https?:\/\/www\.youtube\.com\/embed\/.*$')])
```

Comment:
I see this validator used twice (it also might just be a method, or a callabe, cant remember now)